Add astyle example usage to CONTRIBUTING.md docs

buchanankerswell opened the pull request:
Added example astyle usage to docs.

Similar instructions are in the header of `aspect/contrib/utilities/astyle.rc`, but it is helpful to directly include example astyle usage for single files in `CONTRIBUTING.md`.

bangerth commented on CONTRIBUTING.md:
```diff
@@ -156,6 +156,23 @@ An easy way to install it is through using the following command in Linux (do no
 
+#### Example usage for single files
+
+Correct the indentation of your `.cc` and `.h` files by downloading the ASPECT [style file](https://github.com/geodynamics/aspect/blob/main/contrib/utilities/astyle.rc) and running astyle
```
```suggestion
Correct the indentation of your `.cc` and `.h` files by downloading the ASPECT [style file](https://github.com/geodynamics/aspect/blob/main/contrib/utilities/astyle.rc) and running astyle:
```

bangerth reviewed (commented):
We don't usually run astyle by hand. Rather, we use the `contrib/utilities/indent` script. I think it would be useful to document that instead; it also already has the `astyle.rc` file in that directory.

buchanankerswell commented:
I moved the part about `make indent` from the `Installing astyle` heading to the `Example astyle usage` sub-heading and updated the example usage for individual files.

I also corrected another "to to" typo

buchanankerswell commented:
I corrected a misunderstanding of what `make indent` does. This should be correct now and ready for review. Thanks

bangerth reviewed (approved):
Yes, I like it!
